tests/integration/base_block_test.go

```diff
@@ -154,16 +161,18 @@ func checkPoolDeleted(helper *clients.TestClient, s suite.Suite, name string) {
 	assert.Fail(s.T(), fmt.Sprintf("pool %s was not deleted", name))
 }
 
-func foundPool(helper *clients.TestClient, s suite.Suite, name string) bool {
+func foundPool(helper *clients.TestClient, s suite.Suite, name string) (bool, error) {
 	p := helper.GetPoolClient()
 	pools, err := p.PoolList()
-	assert.Nil(s.T(), err)
+	if err != nil {
+		return false, err
+	}
 	for _, pool := range pools {
 		if name == pool.Name {
-			return true
+			return false, nil
```

Comment:
why isn't this `return true, nil` meaning you found the pool?

Reply:
oops, not sure how that changed